### ISSUE TITLE
Assets and Scripts disappear on page reload

### ISSUE THREAD

**Issue by electricretina** [FluidTYPO3/vhs#1785]
I have just updated to the latest version of VHS 6.1.1 via the TER. My site is running with TYPO3 10.4.27 and PHP 7.4. 

After I clear all caches and load the page everything is rendered correctly and the v:assset.style/script are included in the page source. When I reload the page (CTRL+R) the page reloads without any of the v:asset files. 

It seems like the vhs.asset is not being written to the page cache. 

Page header after I clear caches...
![Screenshot 2022-06-01 124826](https://user-images.githubusercontent.com/1200661/171457915-6af70c4d-cab7-4486-802c-23b06589ff6e.png)

Page header after I reload the page...
![Screenshot 2022-06-01 124534](https://user-images.githubusercontent.com/1200661/171457627-3e1815f1-4da4-4385-9d07-70ac0d1cf3d8.png)

**Comment by electricretina** [comment]
I should have stated this, but thanks for all your work on this extension! 

I have down-graded to 6.0.5 and the described problem goes away.

**Comment by medarob** [comment]
I have the same problem in TYPO3 10.4.30 and PHP 7.4.29.

We use
```
<v:asset.style name="{data.uid}" overwrite="0" group="flex" standalone="0" rewrite="0" namedChunks="0">
... here are some if-then-else-clauses to output css ...
</v:asset.style>
```
in some MASK templates/partials.
VHS 6.0.5 works but after the update to 6.1.0 or 6.1.1 the mentioned error appears. On a page reload the VhsAssetsDependencies are missing in the code. It works only after I cleared the cache.

Reverting back to 6.0.5 seems to solve the problem.

**Comment by NamelessCoder** [comment]
Should be fixed in 6.1.2

**Comment by medarob** [comment]
For me v6.1.2 is working again with the usage of <v:asset.style> after a page reload.
